src/IO/AbstractGeoJSONReader.php

```diff
@@ -0,0 +1,328 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Brick\Geo\IO;
+
+use Brick\Geo\CoordinateSystem;
+use Brick\Geo\Exception\GeometryIOException;
+use Brick\Geo\Geometry;
+use Brick\Geo\GeometryCollection;
+use Brick\Geo\LineString;
+use Brick\Geo\MultiLineString;
+use Brick\Geo\MultiPoint;
+use Brick\Geo\MultiPolygon;
+use Brick\Geo\Point;
+use Brick\Geo\Polygon;
+
+abstract class AbstractGeoJSONReader
+{
+    /**
+     * @param array $geojson
+     * @param int   $srid
+     *
+     * @return Geometry
+     *
+     * @throws \Brick\Geo\Exception\CoordinateSystemException
+     * @throws \Brick\Geo\Exception\UnexpectedGeometryException
+     * @throws GeometryIOException
+     * @throws \Brick\Geo\Exception\InvalidGeometryException
+     */
+    public function readGeoJSON(array $geojson, int $srid) : Geometry
```

Comment:
From what I can see in the [RFC](https://tools.ietf.org/html/rfc7946#section-4), the only supported spatial reference system is WGS 84, so you can remove the parameter and import all geometries with [SRID 4326](https://epsg.io/4326).

Reply:
@BenMorel  I am a bit new to `spatial reference systems SRIDs` ... do you happen to have a link to documentation I can read about these?  I see the References to specific SRIDs in the RFCs, but I am missing context when it comes to the overall idea.

Reply:
SRIDs removed from GeoJson 👍   Please Let me know if I messed something up here... 

Resolved in Commit: https://github.com/brick/geo/pull/14/commits/526ed7fdf5c7aeb88a9eadd9f516700feb0c10d6